ONSdigital/design-system
# Add guidance about the width classes you should apply to specific text inputs like postcode

### What is the context of this PR?
- Updated content and added more detail about making inputs the appropriate width.
- Changed input widths on postcodes to match max width of UK postcodes
- Changed durations and dates macros to use the new `number` type width class

### How to review
- Check input component docs
- Check durations and dates
